models/staging/stg_fivetran_platform__connector.sql

```diff
@@ -25,7 +25,8 @@ final as (
         destination_id,
         connecting_user_id,
         paused as is_paused,
-        signed_up as set_up_at
+        signed_up as set_up_at,
+        coalesce(_fivetran_deleted, {% if target.type == 'sqlserver' %} 0 {% else %} false {% endif %}) as is_deleted
```

Comment:
small suggestion to consolidate the conditional code here.
```suggestion
        coalesce(_fivetran_deleted,{{ ' 0 ' if target.type == 'sqlserver' else ' false'}}) as is_deleted
```